packages-exp/auth-exp/src/model/auth.ts

```diff
@@ -20,45 +20,84 @@ import * as externs from '@firebase/auth-types-exp';
 import { PopupRedirectResolver } from './popup_redirect';
 import { User } from './user';
 
+/** @internal */
 export type AppName = string;
+/** @internal */
 export type ApiKey = string;
+/** @internal */
 export type AuthDomain = string;
 
+/** {@inheritdoc @firebase/auth-types#Config} */
 export interface ConfigInternal extends externs.Config {
+  /**
+   * @internal
+   * @readonly
+   */
   emulator?: {
     url: string;
   };
 }
 
+/** {@inheritdoc @firebase/auth-types#Auth} */
```

Comment:
I think this should just be marked `@internal` in its entirety. No public facing method should accept this type, and users shouldn't know this exists at all